libbeat/tests/system/test_cmd_completion.py

```diff
@@ -17,7 +17,7 @@ def test_bash_completion(self):
     def test_zsh_completion(self):
         exit_code = self.run_beat(extra_args=["completion", "zsh"])
         assert exit_code == 0
-        assert self.log_contains("#compdef mockbeat")
+        assert self.log_contains("#compdef _mockbeat mockbeat")
```

Comment:
Yes, we are not using a fixed version of cobra, and badger depends on a newer version. This version modifies autogenerated completion. I can move the upgrade of cobra to a different PR (actually this could be interesting to do because a 1.0 version has been released).